moonraker/components/update_manager/update_manager.py

```diff
@@ -126,6 +124,18 @@ def __init__(self, config: ConfigHelper) -> None:
         # Auto Status Refresh
         self.refresh_timer: Optional[FlexTimer] = None
         if auto_refresh_enabled:
+            self.refresh_window = config.getintlist('refresh_window', [0, 5],
```

Comment:
This should be moved outside of the condition.  This way a user can toggle auto refresh on or off without removing "refresh_window" in the configuration. 

In addition it guarantees that the `self.refresh_window` attribute is set.  While it shouldn't be reachable when auto refresh is disabled, we avoid potential future bugs if there is a refactor that makes it reachable.

Reply:
Right. I forgot about the warning when an unused option is set. Fixed it now.
I could not decide whether the validation belongs up there with `config.getintlist` or right above the `if auto_refresh_enabled:` section. Let me know if you want it moved again.